src/callbacks_step/analysis_dg2d.jl

```diff
@@ -54,13 +54,33 @@ end
 function calc_error_norms(func, u, t, analyzer,
                           mesh::TreeMesh{2}, equations, initial_condition,
                           dg::DGSEM, cache, cache_analysis)
+  # Calculate error norms for each element
+  l2_errors, linf_errors = calc_error_norms_per_element(func, u, t, analyzer,
+                                                        mesh, equations, initial_condition,
+                                                        dg, cache, cache_analysis)
+
+  # Aggregate element error norms
+  l2_error = sum(l2_errors)
+  linf_error = reduce((x, y) -> max.(x, y), linf_errors)
```

Comment:
Why not use `maximum` along the second dimension (maybe I'm missing something?)
```suggestion
  linf_error = maximum(linf_errors, dims=2)
```

Reply:
`linf_errors` is a `Vector` of `SVector`s, so there is only one dimension. Using `maximum(linf_errors)` instead would lead to different results as `maximum` uses non-broadcasted `max` internally which does not return the componentwise maximum.

Reply:
Got it, thanks